src/tools/illink/src/ILLink.Shared/SharedStrings.resx

```diff
@@ -181,7 +181,7 @@
     <value>There was an unexpected error while trimming. An exception with more details is printed to the MSBuild log. Please share this stack trace with the IL Linker team to further investigate the cause and possible solution.</value>
   </data>
   <data name="LinkerUnexpectedErrorMessage" xml:space="preserve">
-    <value>IL Trimmer has encountered an unexpected error. Please report the issue at https://github.com/dotnet/linker/issues</value>
+    <value>IL Trimmer has encountered an unexpected error. Please report the issue at https://aka.ms/AAj69i2</value>
```

Comment:
Side note (not necessarily for this PR): It's weird that we have a shared error message which is "for the linker". I definitely hope that NativeAOT doesn't use this. Ideally this should be moved to non-shared code (even if we have to duplicate it, since we probably want different message to be shown from NativeAOT)

Reply:
NativeAOT will not make use of the IL1012 since is produced by illink [Driver](https://github.com/dotnet/linker/blob/main/src/linker/Linker/Driver.cs#L836) file (which is not shared with NativeAOT). 
At this moment we don't even have a DiagnosticId range for NativeAOT-specific errors (and I'm not sure that we want one), we could always reuse the warning range IL3050+ to display something similar if needed. 
I still believe that DiagnosticIds should be shared given that most of the warnings can be reused by NativeAOT, and if not used they can just be ignored, illink basically ignores NativeAOT and single file warning Ids.

Reply:
I agree that we should share diagnostic IDs (to a point), my point was about the message string in this case - that ideally we would have a way to specify different message for illink/ilc because they might need to mention the tool by name. But it's not a big problem rigth now.